templates/openstack/what-is-openstack.html

```diff
@@ -666,7 +666,9 @@ <h3>Learn more about OpenStack</h3>
               width="171",
               height="150",
               hi_def=True,
-              loading="lazy" ) |
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img"},
+              ) |
               safe }}
             </div>
             <h4 class="p-heading-icon__title" style="color: #fff">
```

Comment:
With the `is-dark` do we need the online style here? Could you change the text to sentence case as a drive by?

Reply:
There seem to be couple layers of custom styles that override `is-dark` colour from strip. I don't want to mess up too deeply with ubuntu.com styling.